### PR TITLE
Modified to support v4 and v6 aggregare address with summary_only option

### PR DESCRIPTION
**Pull request by Appal5**
##### SUMMARY
-Modified to support both IPv4 and IPv6 Aggregate address commands
- Added support for summary-only option for Aggregate address commands

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
OS10 BGP role

### CODE REVIEW

File: roles/os10_bgp/README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ Role variables
 | ``address_family_ipv4`` | dictionary | Configures IPv4 address family parameters (see ``address_family_ipv4.*``) | os10 |
 | ``address_family_ipv4.aggregate_addr`` | list | Configures IPv4 BGP aggregate entries (see ``aggregate_addr.*``) | os10 |
 | ``aggregate_addr.ip_and_mask`` | string | Configures the IPv4 BGP aggregate address | os10 |
+| ``address_family_ipv4.summary_only`` | boolean | Sets address to summary-only if true | dellos10 |
```
Comment (parkrish) [inline roles/os10_bgp/README.md:40]: Please change it to os10

File: roles/os10_bgp/README.md [hunk 2]
```diff
@@ -45,6 +46,7 @@ Role variables
 | ``address_family_ipv6`` | dictionary | Configures IPv6 address family parameters (see ``address_family_ipv6.*``) | os10 |
 | ``address_family_ipv6.aggregate_addr`` | list | Configures IPv6 BGP aggregate entries (see ``aggregate_addr.*``) | os10 |
 | ``aggregate_addr.ip_and_mask`` | string | Configures the IPv6 BGP aggregate address | os10 |
+| ``address_family_ipv6.summary_only`` | boolean | Sets address to summary-only if true | dellos10 |
```
Comment (parkrish) [inline roles/os10_bgp/README.md:49]: Please change it to os10

File: roles/os10_bgp/README.md (hunk 1 above)
Comment (Appal5) [reply roles/os10_bgp/README.md]: Modified

File: roles/os10_bgp/README.md (hunk 2 above)
Comment (Appal5) [reply roles/os10_bgp/README.md]: Modified to os10

**Pull request merged**